examples/models/population_server.py

```diff
@@ -48,20 +49,24 @@ def pyramid():
     female_quad = Quad(left=0, right="female", bottom="groups", top="shifted", fill_color="#CFF09E")
     female_quad_glyph = plot.add_glyph(source_pyramid, female_quad)
 
-    plot.add_layout(Legend(legends=[("Male", [male_quad_glyph]), ("Female", [female_quad_glyph])]))
+    plot.add_layout(Legend(items=[
+        LegendItem(label="Male", renderers=[male_quad_glyph]),
+        LegendItem(label="Female", renderers=[female_quad_glyph])
+    ]))
```

Comment:
Problem with dict is that it is unordered.  And I don't like the list of dicts because it seems much more complicated and easy to get wrong for no value. Also list of tuples is whats already there it's a smaller step for users to migrate.